cves/CVE-2014-0117.yml

```diff
@@ -114,10 +129,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
-  automated:
-  google:
+  answer: |
+    The vulnerability was found on the 7th of April 2014 by Mark Kroemeke, AKAT-1 and 
+    22733db72ab3ed94b5f8a1ffcde850251fe6f466
```

Comment:
I'd suggest expanding a bit here. Was there a conversation thread somewhere that discussed this? Was there any back and forth? Did they mention what tools or methods they used to discover the vulnerability?